tests/DCPS/Messenger/rtps.ini

```diff
@@ -1,5 +1,8 @@
 [common]
 DCPSGlobalTransportConfig=$file
+DCPSDefaultDiscovery=DEFAULT_RTPS
```

Comment:
Let's not change this file in this PR.  Was this meant to be a local change only?  You can use `rtps_disc.ini` to run a test with rtps discovery.